packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1213,7 +1213,16 @@ export const audiusBackend = ({
   ) {
     const storageV2Enabled = await getFeatureEnabled(FeatureFlags.STORAGE_V2)
     if (storageV2Enabled) {
-      // do storage v2 upload
+      try {
```

Comment:
I added this try-catch here because I removed it from the libs uploadTrackV2 function. I think it was a code smell to have the uploadTrackV2 function wrap its whole body in try-catch